purchase_batch_invoicing/wizards/purchase_batch_invoicing.py

```diff
@@ -92,6 +92,7 @@ def action_batch_invoice(self):
                 invoice.currency_id = po.currency_id
                 invoice.purchase_id = po
                 invoice.purchase_order_change()
+                invoice._onchange_invoice_line_ids()
```

Comment:
Why the onchange? This should be `invoice.compute_taxes` IMO

Reply:
If you think it's better to change it, that's fine with me. I can not prove it, but with the current solution it works well.

Reply:
Yes, because this might done more things than the expected one, depending also on the extension modules. See the patch we have made for v11: #505

Reply:
How to suggest the change in the code? It saw now is possible in Github

Reply:
```suggestion
                invoices.compute_taxes()
```

Reply:
This is BETA TEST

Reply:
Changed.